Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM ubuntu:18.04
+LABEL image=go-translate-server
+
+# Get all the dependencies
+RUN apt update
+RUN apt install -y git autoconf automake libtool curl make g++ unzip
+RUN apt install -y wget git-lfs pkg-config software-properties-common
+RUN git lfs install
+
+# Install protobuf
+RUN cd \
+    && git clone https://github.com/google/protobuf.git \
+    && cd protobuf \
+    && git checkout tags/v3.11.0 \
+    && git submodule update --init --recursive \
+    && ./autogen.sh \
+    && ./configure \
+    && make \
+    && make install \
+    && ldconfig
+
+# Install go
+RUN cd \
+    && wget https://dl.google.com/go/go1.13.linux-amd64.tar.gz \
+    && tar -C /usr/local -xzf go1.13.linux-amd64.tar.gz
+
+ENV PATH="/usr/local/go/bin:${PATH}"
+
+# Install CLD3 for go
+RUN go get github.com/jmhodges/gocld3/cld3
+
+# Install fresh version of cmake (required for building Bergamot translator)
+RUN wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | apt-key add - \
```

Comment:
`wget --https-only --secure-protocol=PFS ...`

Reply:
No longer relevant for `cmake`, but added `--https-only --secure-protocol=PFS` to some other `wget` lines.